### PR TITLE
ver 0.1.0

### PR DESCRIPTION
**Pull request by lunfardo314**
PR before merging into `master` and making `master` default branch

### CODE REVIEW

File: contracts/getproc.go [hunk 1]
```diff
@@ -15,18 +15,18 @@ var (
 	allExamplesMutex = &sync.Mutex{}
 )
 
-// AddProcessor adds new processor to the runtime registry of example processors.
+// AddExampleProcessor adds new processor to the runtime registry of example processors.
 // The 'proc' represents executable of the specific smart contract.
-// It must implement vmtypes.Processor
-func AddProcessor(c *contract.ContractInterface) {
+// It must implement coretypes.Processor
+func AddExampleProcessor(c *coreutil.ContractInterface) {
```
Comment (kape1395) [inline contracts/getproc.go:24]: Not sure, if it is OK to gave ExampleProcessor in a package that is not an example.

File: packages/coretypes/coreutil/contract.go [hunk 2]
```diff
@@ -1,4 +1,6 @@
-package contract
+// package coreutil provides functions to describe interface of the core contract
```
Comment (kape1395) [inline packages/coretypes/coreutil/contract.go:4]: License headers missing.

File: packages/vm/core/accounts/impl.go [hunk 3]
```diff
@@ -3,8 +3,8 @@ package accounts
 import (
 	"fmt"
 	"github.com/iotaledger/wasp/packages/coretypes"
+	assert2 "github.com/iotaledger/wasp/packages/coretypes/assert"
```
Comment (kape1395) [inline packages/vm/core/accounts/impl.go]: Why do we need an alias here?

File: packages/vm/core/blob/impl.go [hunk 4]
```diff
@@ -3,7 +3,7 @@ package blob
 import (
 	"fmt"
 	"github.com/iotaledger/wasp/packages/coretypes"
-	"github.com/iotaledger/wasp/packages/coretypes/coreutil"
+	assert2 "github.com/iotaledger/wasp/packages/coretypes/assert"
```
Comment (kape1395) [inline packages/vm/core/blob/impl.go]: Why the alias is needed here?

File: packages/vm/core/root/impl.go [hunk 5]
```diff
@@ -10,7 +10,7 @@ import (
 	"fmt"
 	"github.com/iotaledger/goshimmer/dapps/valuetransfers/packages/balance"
 	"github.com/iotaledger/wasp/packages/coretypes"
-	"github.com/iotaledger/wasp/packages/coretypes/coreutil"
+	assert2 "github.com/iotaledger/wasp/packages/coretypes/assert"
```
Comment (kape1395) [inline packages/vm/core/root/impl.go:13]: The same here.

File: packages/vm/core/testcore/sandbox_tests/test_sandbox_sc/impl_check_ctx.go [hunk 6]
```diff
@@ -2,7 +2,7 @@ package test_sandbox_sc
 
 import (
 	"github.com/iotaledger/wasp/packages/coretypes"
-	"github.com/iotaledger/wasp/packages/coretypes/coreutil"
+	assert2 "github.com/iotaledger/wasp/packages/coretypes/assert"
```
Comment (kape1395) [inline packages/vm/core/testcore/sandbox_tests/test_sandbox_sc/impl_check_ctx.go]: and here

File: plugins/banner/plugin.go [hunk 7]
```diff
@@ -11,7 +11,7 @@ const PluginName = "Banner"
 
 const (
 	// AppVersion version number
-	AppVersion = "v0.0.1"
+	AppVersion = "v0.1.0"
```
Comment (kape1395) [inline plugins/banner/plugin.go:14]: Maybe it should be taken from `git describe --tags` automatically?

Comment (kape1395) [reply plugins/banner/plugin.go:14]: One way to do that: https://stackoverflow.com/questions/11354518/application-auto-build-versioning

**Review by kape1395 — COMMENTED** [review]
Comments are mainly on naming.

File: packages/coretypes/coreutil/contract.go (hunk 2 above)
Comment (lunfardo314) [reply packages/coretypes/coreutil/contract.go:4]: fixed

File: contracts/getproc.go (hunk 1 above)
Comment (lunfardo314) [reply contracts/getproc.go:24]: essentially all those contracts are examples. Cannot come to better categorization atm

File: packages/vm/core/accounts/impl.go (hunk 3 above)
Comment (lunfardo314) [reply packages/vm/core/accounts/impl.go]: no need. Just artifact of refactoring. Fixed

File: packages/vm/core/blob/impl.go (hunk 4 above)
Comment (lunfardo314) [reply packages/vm/core/blob/impl.go]: fixed

File: plugins/banner/plugin.go (hunk 7 above)
Comment (lunfardo314) [reply plugins/banner/plugin.go:14]: I know, looked at it, seems complicated, postponed for the future as TODO

**Review by kape1395 — APPROVED** [review]

**Pull request merged**